queries/leo/highlights.scm

```diff
@@ -173,11 +173,10 @@
   (identifier) @variable.parameter)
 
 (struct_declaration
-  name: (identifier) @variable.member)
+  name: (identifier) @type.definition)
```

Comment:
This gets overridden by `@constant` if the name is capitalized, is this intended? Same with records

Reply:
Unless that is an official part of the grammar (in which case the parser should handle it), we should stop this sort of case-dependent convention captures.

Reply:
Nope, it is part of my learning curve! Fixing it now.

Reply:
Guys I think I did the necessary changes, will resolve this conversation.